src/libmain/progress-bar.cc

```diff
@@ -365,6 +368,62 @@ class ProgressBar : public Logger
         writeToStderr("\r" + filterANSIEscapes(line, false, width) + ANSI_NORMAL + "\e[K");
     }
 
+		void draw_multi(State & state)
+		{
+			state.haveUpdate = false;
+			if (!state.active) return;
+
+			std::string line;
+			auto width = getWindowSize().second;
+			if (width <= 0) width = std::numeric_limits<decltype(width)>::max();
+
+			std::string status = getStatus(state);
+			if (!status.empty()) {
+				line += '[';
+				line += status;
+				line += "]";
+			}
+
+      #if defined(WIN32) || defined(_WIN32) || defined(__WIN32__) || defined(__NT__)
+			  system("cls");
+			#else
```

Comment:
Since Nix uses spaces for indentation, please use spaces (instead of tabs) here as well to stay consistent.